src/plonetheme/tokyo/browser/overrides/cmfplone/Products.CMFPlone.resources.browser.scripts.pt

```diff
@@ -36,3 +36,19 @@
 
   </tal:block>
 </tal:scripts>
+
+<script>
+  define = undefined;
+  require = undefined;
+
+</script>
+
+<script type="text/javascript" src="++plone++plonetheme.tokyo/moment.min.js"></script>
+<script type="text/javascript" src="++plone++plonetheme.tokyo/popper.min.js"></script>
+<script type="text/javascript" src="++plone++plonetheme.tokyo/bootstrap.min.js"></script>
+
+<script>
+  define = _old_define;
+  require = _old_require;
+
```

Comment:
i think this is resetting the wrong _old_define, because it is ignoring the define-changes of the last plone-script... 
`old_define` should be set in line 41. Same for `old_require`